pkg/asset/manifests/aws/cloudproviderconfig.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/pkg/errors"
+	ini "gopkg.in/ini.v1"
+
+	awstypes "github.com/openshift/installer/pkg/types/aws"
+)
+
+// CloudConfig is the aws cloud provider config.
+type CloudConfig struct {
+	Global global
+}
+
+// global struct of CloudConfig which is currently not inialized.
+type global struct {
```

Comment:
Did we determine if this struct is needed or not? What is the actual output in the cloud provider config considering that we're not populating any of the values?

Reply:
Update the Global struct and removed unused attributes.

Reply:
Removed all the unused attributes in the global struct.